### PR TITLE
Add state timestamp

### PR DESCRIPTION
**Pull request by bolahanna44**
am afraid we will keep adding columns as we introduce more states

### CODE REVIEW

**Comment by bolahanna44** [comment]
not sure why rspec stage is failing @jozefvaclavik any thoughts ?

**Comment by jozefvaclavik** [comment]
Can you drop the `uploaded` state? Coz while that works for an import, it doesnt really work through export flow.

**Comment by jozefvaclavik** [comment]
One more thing. This https://github.com/sandbite/perilune/pull/1/files#diff-194f3123249c993e39d33f14a8f60f48dc54b39938b994f9a282c9e996a0f6d5R7 introduces `started` state instead of `uploaded`.

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -34,27 +31,34 @@ jobs:
...
         run: sudo apt-get install libpq-dev
-      - name: Setup app dependencies
-        run: |
-          gem install bundler -v 1.17.3 --no-document
-          bundle install --jobs 4 --retry 3
       - name: Setup Database
+
+        run: |
+          cp config/github_database.yml config/database.yml
+          bundle exec rails db:create db:migrate
```
Comment (jozefvaclavik) [inline .github/workflows/ruby.yml]: Usually for CI its better to do `db:schema:load` instead of `db:migrate`.

**Pull request merged**